tests/e2e/integrations/cut_cross_entropy.py

```diff
@@ -0,0 +1,62 @@
+"""
+Simple end-to-end test for Cut Cross Entropy integration
+"""
+
+import unittest
+from pathlib import Path
+
+from axolotl.cli import load_datasets
+from axolotl.common.cli import TrainerCliArgs
+from axolotl.train import train
+from axolotl.utils.config import normalize_config
+from axolotl.utils.dict import DictDefault
+
+from ..utils import with_temp_dir
+
+
+class CutCrossEntropyIntegrationTestCase(unittest.TestCase):
+    """
+    e2e tests for cut_cross_entropy integration with Axolotl
+    """
+
+    @with_temp_dir
```

Comment:
this decorator can be removed. `temp_dir` should be an automatic pytest fixture.

Reply:
```
FAILED tests/e2e/integrations/cut_cross_entropy.py::CutCrossEntropyIntegrationTestCase::test_llama_w_cce - TypeError: CutCrossEntropyIntegrationTestCase.test_llama_w_cce() missing 1 required positional argument: 'temp_dir'
```

It would error out if I remove the decorator.

Reply:
change it so the test class doesn't extend unittest.TestCase, i.e. `class CutCrossEntropyIntegrationTestCase:`